Gemfile

```diff
@@ -1,15 +1,18 @@
 source 'https://rubygems.org'
 
 # install correct ruby version with rbenv, which can be installed with brew install rbenv
-ruby '2.7.6'
+ruby '3.1.0'
+
+gem 'bundler', '~> 2.6.5'
 
 gem "fastlane"
 gem "dotenv"
-# Cocoapods 1.15 introduced a bug which break the build. We will remove the upper
-# bound in the template on Cocoapods with next React Native release.
-gem 'cocoapods', '>= 1.13', '< 1.15'
-gem 'activesupport', '>= 6.1.7.5', '< 7.1.0'
-gem "configure_extensions"
```

Comment:
@jorelosorio The build is failing for the OMS now that `configure_extensions` is removed. Could you have a look? https://github.com/AtB-AS/mittatb-app/actions/runs/13834854603/job/38707329323